esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
> Btw seems github now puts badges next to names in commit comments: pfalcon is 'MicroPython member' while dpgeorge and me are labelled 'contributor'. What's up with that?

Yup, and I love that very much. Both dpgeorge and myself are 'MicroPython member' for me.